lighthouse-core/audits/server-response-time.js

```diff
@@ -11,27 +11,26 @@ const MainResource = require('../computed/main-resource.js');
 
 const UIStrings = {
   /** Title of a diagnostic audit that provides detail on how long it took from starting a request to when the server started responding. This descriptive title is shown to users when the amount is acceptable and no user action is required. */
-  title: 'Server response times are low (TTFB)',
+  title: 'Initial server response time was short',
```

Comment:
went with 'short' but if this is looking too long it could be `'Initial server response was quick'`, but then it starts drifting from the failureTitle (and I start getting confused about them being from the same audit if they get too different :) so then we might have to update that as well...